removal_from_ansible.rst

```diff
@@ -119,8 +119,22 @@ Process
 Unmaintained collections
 ========================
 
-Identifying and removing an unmaintained collection
----------------------------------------------------
+Collections that have been announced deprecated / unmaintained by their maintainers
```

Comment:
```suggestion
Removing a collection that has been explicitly deprecated or abandoned by its former maintainers
```

Reply:
I don't like "former maintainers" here. When they deprecate or abandon the collection, they are still the maintainers. They are only _former maintainers_ afterwards.

What I mean is: Only _current_ maintainers can deprecate or abandon a collection. What do you think about my [last commit](https://github.com/ansible-collections/overview/pull/214/commits/81c1150dc3970d46f23c503e0e9822db804fd36a)?

Reply:
It's not a big deal either way, but when this process is taking place they are the former maintainers so it feels weird to call them the maintainers. The collection currently has no maintainers, or else it would not be eligible for this process.

Reply:
@Andersson007 @briantist @felixfontein @russoz @samccann What do you think, "former maintainers" or just "maintainers"? Personally, I don't like the "former" but I'm open to both.

Reply:
Deprecated collections can still be maintained. Only when they are unmaintained or abandoned (is there actually a difference between the two?) they no longer have maintainers. So depending on context, both can be the right wording. So how about writing `(former) maintainers`?